test/test_data.js

```diff
@@ -227,7 +227,7 @@ const tests = {
       });
   },
 
-  test_multiframe_us: () => {
```

Comment:
why is it commented out now? Does it not work?

Reply:
yes it is failing:

(node:39227) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/home/davide/Development/OIHF/dcmjs/test/cine-test.dcm'

and I don't see any dcmjs/test/cine-test.dcm file in the repo